### PR TITLE
Add docs about additions

### PR DESCRIPTION
**Pull request by herostrat**
I don't know if this is useful for others and ok for you blake.
Background: I only recently found out about doubletake and thought to make it easier to find useful additions.

If this should not go in the docs, no harm done, you can just close it.

### CODE REVIEW

**Comment by NickM-27** [comment]
Can you point this at master since it's just a docs update?

I think it would be good to emntion this, `Third Party Integrations` might be a better name but I think it makes sense.

**Comment by herostrat** [comment]
Sure thing, I will change it to master.

I changed it to "Third Party Extensions" as Integrations is kind of loaded with home-assistant.

**Comment by NickM-27** [comment]
> Sure thing, I will change it to master.
> 
> I changed it to "Third Party Extensions" as Integrations is kind of loaded with home-assistant.

Yeah that's fair, sounds good to me

**Comment by blacknell** [comment]
Couple of spelling mistakes - being not beiung; an overview of additions

**Comment by blacknell** [comment]
> Couple of spelling mistakes - being not beeing; an overview of additions

@herostrat @NickM-27 - I can't fix these myself as I'm not marked as a repo contributor

**Comment by herostrat** [comment]
Thanks for the review. I will fix them after work today.

**Comment by blacknell** [comment]
Could I suggest moving the hyperlink to double take from the heading to the body of text. It's more consistent visually and wasn't immediately obvious.

**Comment by NickM-27** [comment]
@blacknell you should be able to make suggestions / comments directly on the code by opening "view files" and then clicking on a line

File: docs/docs/integrations/third_party_extensions.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Beeing open source, others have the possibility to modify and extend the rich functionality Frigate already offers.
+This page is meant to be a overview over additions one can make to the home NVR setup. The list is not exhaustive and can be extended via PR to the Frigate docs.
+
+:::caution
+This page does not recommend or rate the presented projects.
+Please use your own knowledge to assess and vet them before you install anything on your system.
+
+:::
+
+## [Double Take](https://github.com/jakowenko/double-take)
```
Comment (blacknell) [inline docs/docs/integrations/third_party_extensions.md:15]: Suggest move the URL from the heading to the paragraph text to be visually consistent with the rest of the documentation (and more obvious)

File: docs/docs/integrations/third_party_extensions.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+id: third_party_extensions
+title: Third Party Extensions
+---
+
+Beeing open source, others have the possibility to modify and extend the rich functionality Frigate already offers.
```
Comment (blacknell) [inline docs/docs/integrations/third_party_extensions.md:6]: [spelling] Being

File: docs/docs/integrations/third_party_extensions.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+This page is meant to be a overview over additions one can make to the home NVR setup. The list is not exhaustive and can be extended via PR to the Frigate docs.
```
Comment (blacknell) [inline docs/docs/integrations/third_party_extensions.md:7]: change a to an (an overview)

**Review by blacknell — CHANGES_REQUESTED** [review]

File: docs/docs/integrations/third_party_extensions.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+Being open source, others have the possibility to modify and extend the rich functionality Frigate already offers.
+This page is meant to be an overview over additions one can make to the home NVR setup. The list is not exhaustive and can be extended via PR to the Frigate docs.
+
+:::caution
+This page does not recommend or rate the presented projects.
+Please use your own knowledge to assess and vet them before you install anything on your system.
+
+:::
```
Comment (NickM-27) [inline docs/docs/integrations/third_party_extensions.md:13]:
```suggestion
:::caution

This page does not recommend or rate the presented projects.
Please use your own knowledge to assess and vet them before you install anything on your system.

:::
```

File: docs/docs/integrations/third_party_extensions.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+## [Double Take](https://github.com/jakowenko/double-take)
+
+[Double Take](https://github.com/jakowenko/double-take) provides an unified UI and API for processing and training images for facial recognition.
```
Comment (NickM-27) [inline docs/docs/integrations/third_party_extensions.md:17]: Might be worth noting here that doubletake supports automatically setting the sub labels for `Person` objects that are detected and recognized

**Comment by herostrat** [comment]
Not stale, but I don't know if it is desired

**Review by NickM-27 — APPROVED** [review]

**Comment by skrashevich** [comment]
This project does not seem to be supported anymore. I have a live [fork](https://github.com/skrashevich/double-take). Maybe it makes sense to replace the link?

**Pull request merged**